#163388096 Add Tests for the password reset feature

asirvex opened the pull request:
### What does this PR do?
- Add tests for password reset actions and reducer
- Add tests for password update actions and reducer

### Description of Task to be completed?
- Add tests for password reset actions and reducer
- Add tests for password update actions and reducer

### How should this be manually tested?

- Clone the github repository
`git clone https://github.com/andela/ah-the-phoenix-frontend.git`
- Change your working directory to the project folder
`cd ah-fulldeck-frontend`
- Checkout to thebranch ft-password-reset-163388096
`git checkout ft-password-reset-163388096 `
- Install the required dependencies using
` npm install`
-Run tests
` ./node_modules/.bin/jest`

### Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
[#163388096](https://www.pivotaltracker.com/story/show/163388096)

ezrqnkemboi commented on src/views/getProfile.js:
```diff
@@ -12,15 +12,14 @@ export class Profile extends Component {
-    const { fetched } = this.props;
-    if (!fetched) {
+    const { isFetching } = this.props;
+    if (isFetching === true) {
```
If you just say, isfetching, then it will definately translate to true. No need for ```== true```

ezrqnkemboi commented on src/views/editProfile.js:
```diff
@@ -1,8 +1,23 @@
+  }
   render() {
+    const { isFetching } = this.props;
+    if (isFetching === true) {
```
Just check on ```isFetching```, no need for checking with an equal operator.

asirvex replied:
I was not doing the profile feature, I think the comments are for Kevin's feature

ezrqnkemboi replied:
Yes, i think, @Arusey work should be merged, then you get the latest changes and rebase them.

asirvex committed:
Chore(password reset): write tests for the feature

- add tests for the password reset actions and reducers
- add tests for the password update actions and reducers

[Deliver #163388096]
